examples/influxdb_data_model.ipynb

```diff
@@ -36,7 +36,7 @@
     "\n",
     "* lsst.verify package -> InfluxDB measurement\n",
     "* lsst.verify metadata -> InfluxDB tag\n",
-    "* lsst.verify metric value - > InfluxDB field\n",
+    "* lsst.verify metric -> InfluxDB field\n",
```

Comment:
Should this line say "InfluxDB field key" (if I understand the [InfluxDB terminology](https://docs.influxdata.com/influxdb/v1.7/concepts/key_concepts/#discussion) correctly), and should there be another line saying `lsst.verify metric value -> InfluxDB field value`?

Reply:
Also, please tell me this notebook isn't the primary documentation for the translations...

Reply:
No, this notebook is to exercise the mapping of lsst.verify jobs to InlfuxDB and it needs to be properly documented.